tests/simple/testdata/macros.textproto

```diff
@@ -1,2 +1,50 @@
 name: "macros"
 description: "Tests for CEL macros."
+section {
+  name: "exists"
+  description: "Checks for the existence of a map key or list element"
+  test {
+    name: "list_elem_exists"
+    expr: "[1, 2, 3].exists(e, e == 2)"
+    value: { bool_value: true }
+  }
+  test {
+    name: "not_list_elem_exists"
+    expr: "![1, 2, 3].exists(e, e > 3)"
+    value: { bool_value: true }
+  }
+  test {
+    name: "list_elem_mixed_type_exists"
+    expr: "[1, '2', 3].exists(e, e != '3')"
+    value: { bool_value: true }
+  }
+  test {
+    name: "list_elem_mixed_type_exists_error"
+    expr: "[1, '2', 3].exists(e, e == '3')"
+    eval_error: {
+      errors: { message: "no such overload" }
+    }
+  }
+  test {
+    name: "map_key_exists"
+    expr: "{'a':1, 'b':2}.exists(k, k == 'b')"
+    value: { bool_value: true }
+  }
+  test {
+    name: "not_map_key_exists"
+    expr: "!{'a':1, 'b':2}.exists(k, k == 'c')"
+    value: { bool_value: true }
+  }
+  test {
+    name: "map_key_mixed_type_exists"
```

Comment:
How about renaming this to `map_value_mixed_type_exists` and replacing `k` with `v`?